.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,15 +23,15 @@ Change [ ] to [X] to leave a check-mark
 -->
 
 - [ ] Local unit tests
-- [ ] Local integration tests
-- [ ] Environment system tests
+- [ ] Local integration tests (i.e. Component Tests) 
+- [ ] Environment system tests (i.e. Integration Tests) 
 - [ ] Tested in production
 
 <!--
 ## Checklist
 
 - [ ] JIRA Issue(s) ID in PR title and commits
-- [ ] I have rebased my changes onto the head of the target branch before opening this PR
+- [ ] I have resolved any merge conflicts with the head of the target branch before opening this PR
```

Comment:
because we squash anyway,theres no difference between rebase or merge on change branch